README.md

```diff
@@ -14,9 +14,9 @@ Mirador uses [Node.js](https://nodejs.org/) and a build system to assemble, test
  1. Install [Node.js](https://nodejs.org/)
  2. Install the Grunt command line runner i.e. `npm install -g grunt-cli`
  3. Install the Bower command line utility i.e. `npm install -g bower`
- 1. Clone the Mirador repository
- 1. Change into the Mirador directory
- 1. Install all dependencies with `npm install`. Run `npm start`.
+ 4. Clone the Mirador repository
```

Comment:
Can we leave these all as 1's please? That way if one above gets updated they all don't need to be reordered?